<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mostrador</class>
 <widget class="QWidget" name="Mostrador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mostrador</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Fabricante:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="etiqFabricante">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Producto:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="etiqProducto">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nro. de Serie:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="etiqNroSerie">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Indice:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="etiqIndice">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>16</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CharlaToken</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="editorLineaEnvio">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="botonEnvio">
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Dato Recibido:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editorLineaRecibida"/>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="cajaTextoDatosFlash">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonLeerFlash">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Leer Flash</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="barraProgreso">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>label_6</zorder>
  <zorder>cajaTextoDatosFlash</zorder>
  <zorder>botonLeerFlash</zorder>
  <zorder>barraProgreso</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
